src/services/doppler-billing-api-client.ts

```diff
@@ -79,6 +81,8 @@ export class HttpDopplerBillingApiClient implements DopplerBillingApiClient {
       accountId: x.accountId,
       product: x.product,
       date: new Date(x.date),
+      creationDate: !!x.creationDate ? new Date(x.creationDate) : null,
```

Comment:
Tendría sentido agregar el fallback para que se siga mostrando una fecha si aún no se aplicaron los cambios en la API?

```suggestion
      creationDate: !!x.creationDate ? new Date(x.creationDate) : new Date(x.date),
```

Reply:
me parece bien, si el creationDate es nulo que le ponga la otra fecha. Joya

Reply:
Done